packages/main/src/features/InputElementsFormSupport.js

```diff
@@ -26,6 +26,41 @@ class FormSupport {
 		}
 	}
 
+	static syncNativeVisibleInput(element, nativeInputUpdateCallback, nativeInputChangeCallback) {
+		const needsNativeInput = !!element.name;
+		let nativeInput = element.querySelector(`input[type=${element._type || "hidden"}][data-ui5-form-support]`);
+
+		if (needsNativeInput && !nativeInput) {
+			nativeInput = document.createElement("input");
+			nativeInput.type = element._type;
```

Comment:
Hardcode this (and ._type will be unnecessary in the FileUploader.js then)

Reply:
It is not a good idea to hardcode it, because it is used as well on line 31.